src/features/network/store.ts

```diff
@@ -19,8 +24,28 @@ interface NetworkActions {
 
 export const useNetworkStore = create<NetworksState & NetworkActions>(
   persist(
-    set => ({
+    (set, get) => ({
       ...initialState,
+      getNetworks: () => {
+        const hostname = window.location.hostname
+        const networks = Array.from(get().byId)
+          .map(([id, network]) => ({ id, ...network }))
+          .sort(({ name: a }, { name: b }) =>
+            a.toLowerCase() < b.toLowerCase() ? -1 : 1,
+          )
+          .filter(
+            ({ filter }) =>
+              !filter ||
+              hostname.includes(filter) ||
+              hostname.includes("localhost"),
+          )
+        return networks
+      },
+      getActiveNetwork: () => {
+        const networks = get().getNetworks()
+        const activeNetwork = networks.find(({ id }) => id === get().activeId)
+        return activeNetwork ?? networks[0]
```

Comment:
It's possible that the "active network" in your browser cache no longer exists, in which case fall back to the first one alphabetically.